### DISCUSSION TITLE
How can I find the controltemplate to the control?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#10901] [Q&A] [answered]
In previous topic I need to achieve what I need by the DropDownButton :
#https://github.com/AvaloniaUI/Avalonia/discussions/10896

Whereas, some styles of DropDownButton differ from ComboBox much.

![1](https://user-images.githubusercontent.com/30720036/229727234-f0ede32f-fb0d-45cc-82fa-abfde530529f.gif)

As you can see, the two controls behave differently when the mouse is over.

I want the DropDownButton to behave the same as the combobox when mouse is over.

It seems I have to rewrite the ControlTemplate of DropDownButton for this.

Em.. in WPF there is a sample ControlTemplate tutorial of each control. For example: https://learn.microsoft.com/en-us/dotnet/desktop/wpf/controls/combobox-styles-and-templates?view=netframeworkdesktop-4.8

In Avalonia it just has a code-behind source code but not a ControlTemplate. Without the ControlTemplate I don't know how to start my job.

Where I can find the ControlTemplate of DropDownButton? Or is there any way I can achieve this? Thank you.

**Comment by timunie** [comment, marked as answer]
You are lucky, as Avalonia is open source. So just lookup the original styles. I assume you use v11-p6 and have the Fluent-theme isntalled. Here you go: 

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/DropDownButton.xaml

**Reply by kyuranger** [reply]
> You are lucky, as Avalonia is open source. So just lookup the original styles. I assume you use v11-p6 and have the Fluent-theme isntalled. Here you go:
> 
> https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/DropDownButton.xaml

Thanks a lot. That\s great. This ControlTemplate URL should be added on the introduction page of each control (eg https://docs.avaloniaui.net/docs/controls/combobox)

**Reply by timunie** [reply]
yeah probably in docs for v11.0

**Reply by kyuranger** [reply]
> yeah probably in docs for v11.0

Thanks for your hard work.
